src/pages/references/document-sandbox/document-apis/classes/Node.md

```diff
@@ -188,111 +156,150 @@ The node's parent. Undefined if the node is an orphan, or if the node is the art
 
 ___
 
-### relativeRotation
+### rotation
 
-• `get` **relativeRotation**(): `number`
+• `get` **rotation**(): `number`
 
-The node's local rotation value in degrees, relative to its parent's axes. Modifying this value will also adjust the
-node's x & y translation such that the node's center is in the same location after the rotation – i.e. this setter
-rotates the node about its bounding box's center, not its origin.
+The node's local rotation angle in degrees, relative to its parent's axes. Use `setRotationInParent` to
+change rotation by rotating around a defined centerpoint.
 
 #### Returns
 
 `number`
 
-• `set` **relativeRotation**(`value`): `void`
+___
 
-#### Parameters
+### rotationInScreen
 
-| Name | Type |
-| :------ | :------ |
-| `value` | `number` |
+• `get` **rotationInScreen**(): `number`
+
+The node's total rotation angle in degrees, relative to the overall global view of the document – including any
+cumulative rotation from the node's parent containers.
 
 #### Returns
 
-`void`
+`number`
 
 ___
 
-### relativeTransform
+### transformMatrix
 
-• `get` **relativeTransform**(): [`mat2d`](https://glmatrix.net/docs/module-mat2d.html)
+• `get` **transformMatrix**(): `https://glmatrix.net/docs/module-mat2d.html`
 
 The node's transform matrix relative to its parent.
 
 #### Returns
 
-[`mat2d`](https://glmatrix.net/docs/module-mat2d.html)
+`https://glmatrix.net/docs/module-mat2d.html`
```

Comment:
Looks like somehow we lost the clickable links here. Compare to the markup for the old getter:


```md
### relativeTransform

• `get` **relativeTransform**(): [`mat2d`](https://glmatrix.net/docs/module-mat2d.html)

The node's transform matrix relative to its parent.

#### Returns

[`mat2d`](https://glmatrix.net/docs/module-mat2d.html)
```

Reply:
@peterflynn @ArGup fixed all the links and committed back.